src/module/rolls/chat-message.ts

```diff
@@ -260,21 +274,15 @@ export class IronswornRollChatMessage {
     if (c1 === undefined || c2 === undefined) return {}
 
     const momentum = this.actor.data.data.momentum
+    const rawOutcome = this.roll.rawOutcome?.value
     const momentumOutcome = computeRollOutcome(momentum, c1, c2)
 
-    // compare this.roll.rawOutcome?.value
-    // and momentumOutcome
-
-    switch (
-      momentumBurnWouldUpgrade(this.roll.rawOutcome?.value, momentumOutcome)
-    ) {
-      case true:
-        return {
-          possibleMomentumBurn: outcomeText(momentumOutcome, c1 === c2),
-        }
-      default:
-        return {}
+    if (!isUndefined(rawOutcome) && momentumOutcome > rawOutcome) {
```

Comment:
I know it's not much shorter, but wouldn't it still make sense to use `momentumBurnWouldUpgrade` here? For clarity of intent if nothing else.

Reply:
oddly enough, originally i kept it as its own `momentumBurnWouldUpgrade`. i'm open to going back to that, but here's my rationale for its current state:

* the only code inside `momentumBurnWouldUpgrade` was `return momentumOutcome > rawOutcome`, which seemed very lonely :pensive:
* another possibility i considered was renaming `momentumBurnWouldUpgrade` to something that expressed what it *does* (comparing two different move outcomes, which might have other applications) rather than what it's *used by*

the thought experiment that clinched it for me was: if i was unaware of the numeric move outcome enum, but broadly understood the game and what "outcome" referred to, what would i make of `momentumOutcome > rawOutcome`? 

the conclusion i reached on that was: i'd probably get the jist of "this is a comparison of two move outcomes, one from momentum and 'raw' one (whatever that is), to find out if one is higher than the other". big YMMV on that, though.

on the other hand, i reckon there's some value in functions beyond the code within them - like, it also abstracts the code into a human-friendly "label" on what's happening... and it can serve as an attachment point for documentation.

how about this: i give it a pass for more annotation via comments like youve already established within other functions, breaking it down step by step. if that looks OK, that might be the best of both worlds. if it doesn't, i can put it in a function again and we'll still have the annotations.